examples/dtls-echo/Makefile

```diff
@@ -49,6 +49,8 @@ CFLAGS += -DDEVELHELP
 # NOTE: Add the package for TinyDTLS
 USEPKG += tinydtls
 
+USEMODULE += tinydtls
```

Comment:
Why is this needed for this package, but not for other packages (e.g., ccn-lite) then?